bika/lims/browser/js/coffee/bika.lims.analysisrequest.add.coffee

```diff
@@ -541,11 +541,16 @@ class window.AnalysisRequestAdd
      * Apply the sample data to all fields of arnum
     ###
 
-    # set the sampling date (required)
+    # set the sampling date
     field = $("#SamplingDate-#{arnum}")
     value = sample.sampling_date
     field.val value
 
+    # set the date sampeld
+    field = $("#DateSampled-#{arnum}")
+    value = sample.date_sampled
+    field.val value
+
```

Comment:
Try to keep your commits consistent. E.g. this change is independent of the hook you introduced above

Reply:
Got it